Formula/octave.rb

```diff
@@ -141,14 +141,18 @@ def install
     # This is supposed to crash octave if there is a problem with BLAS
     system bin/"octave", "--eval", "single ([1+i 2+i 3+i]) * single ([ 4+i ; 5+i ; 6+i])"
     # Test basic compilation
-    (testpath/"oct_demo.cc").write("#include <octave/oct.h>\n"\
-                                   "DEFUN_DLD (oct_demo, args, /*nargout*/, \"doc str\")"\
-                                   "{ return ovl (42); }\n")
+    (testpath/"oct_demo.cc").write <<~EOS
+      #include <octave/oct.h>
+      DEFUN_DLD (oct_demo, args, /*nargout*/, "doc str")
+      { return ovl (42); }
+    EOS
     system bin/"octave", "--eval", "mkoctfile('oct_demo.cc'); assert(oct_demo, 42)"
     # Test FLIBS environment variable
-    system bin/"octave", "--eval", "args = strsplit (mkoctfile ('-p', 'FLIBS')); "\
-                                   "args = args(~cellfun('isempty', args));"\
-                                   "mkoctfile (args{:}, 'oct_demo.cc'); "\
-                                   "assert(oct_demo, 42)"
+    system bin/"octave", "--eval", <<~EOS
+      args = strsplit (mkoctfile ('-p', 'FLIBS'));
+      args = args(~cellfun('isempty', args));"
+      mkoctfile (args{:}, 'oct_demo.cc'); "
```

Comment:
```suggestion
      args = args(~cellfun('isempty', args));
      mkoctfile (args{:}, 'oct_demo.cc');
```
Are the `"`s still needed?

Reply:
It is not needed anymore.